pkg/controller/netflow.go

```diff
@@ -106,44 +107,38 @@ func (cont *AciController) queueNetflowUpdate(netflowpolicy *netflowpolicy.Netfl
 	cont.netflowQueue.Add(key)
 }
 
-func (cont *AciController) netflowPolicyDelete(obj interface{}) {
+func (cont *AciController) netflowPolicyDelete(obj interface{}) bool {
 	nf, isNf := obj.(*netflowpolicy.NetflowPolicy)
 	if !isNf {
 		deletedState, ok := obj.(cache.DeletedFinalStateUnknown)
 		if !ok {
 			NetflowPolicyLogger(cont.log, nf).
 				Error("Received unexpected object: ", obj)
-			return
+			return false
 		}
 		nf, ok = deletedState.Obj.(*netflowpolicy.NetflowPolicy)
 		if !ok {
 			NetflowPolicyLogger(cont.log, nf).
 				Error("DeletedFinalStateUnknown contained non-netflow object: ", deletedState.Obj)
-			return
+			return false
 		}
 	}
 	nfkey, err := cache.MetaNamespaceKeyFunc(nf)
 	if err != nil {
 		NetflowPolicyLogger(cont.log, nf).
 			Error("Could not create netflow key: ", err)
-		return
+		return false
 	}
 	cont.apicConn.ClearApicObjects(cont.aciNameForKey("nfp", nfkey))
+	return false
```

Comment:
same comment...you're always returning false regardless of whether there's a problem